coffea/processor/executor.py

```diff
@@ -888,37 +862,21 @@ def _work_function(item, processor_instance, flatten=False, savemetrics=False,
                 'fileuuid': str(uuid.UUID(bytes=item.fileuuid))
             }
             with filecontext as file:
-                if schema is None:
-                    # To deprecate
```

Comment:
I see LazyDataFrame is still in the source. I think it does cover some use cases where a file is just not readable via NanoEvents with base schema presently. Perhaps the better solution is to make https://github.com/CoffeaTeam/coffea/blob/master/coffea/nanoevents/mapping.py#L195 more robust, but in the meantime any strong motivation not to leave the LazyDataFrame (via setting schema to None) ?

Reply:
I'll make None give you LazyDataFrame, but default to BaseSchema.

Reply:
Tagging this for rc2